docs/usage.rst

```diff
@@ -162,6 +162,69 @@ mapping that is checked against the :ref:`schema <schema_dict-rule>` rule:
    ``allow_unknown`` can also be set to a validation schema.
 
 
+.. _requiring-all:
+
+Requiring all
+-------------
+See also :ref:`this paragraph <require_all>`, :ref:`this paragraph <required>`.
+
+By default any keys defined in the schema are not required:
+
+.. doctest::
+
+    >>> schema = {'name': {'type': 'string', 'maxlength': 10}}
+    >>> v.validate({}, schema)
+    True
+
+However, you can require all document keys pairs by setting
```

Comment:
i think that suffices as a sentence after the explanation of the variant when it is set at initialization. an inline example (`validator_require_all = True`) should also be enough, no need for a code block.